app/views/split_checkout/_summary.html.haml

```diff
@@ -69,7 +69,7 @@
     = render 'spree/orders/summary', order: @order, display_footer: false
 
 
-.summary-right
+.summary-right{ "data-controller": "sticky", "data-sticky-target": "container" }
```

Comment:
I assume this would apply to all screen-sizes?

Reply:
Yep, but this is ok because of the surrounding css (parent and sibling)